### DISCUSSION TITLE
Deployer recipes in dist version

### DISCUSSION THREAD

**Discussion by aidavia** [deployphp/deployer#2817] [Help needed]
Watching this repo https://github.com/deployphp/distribution i cant see any recipe/contrib folder. How can i use these? Is it a different package to install?

**Comment by pokgak** [comment]
The recipes are in the `recipes` folder and 3rd-party recipes are now directly in this repo under the [`contrib`](https://github.com/deployphp/deployer/tree/master/contrib) folder.

**Comment by aidavia** [comment]
@pokgak But there is no recipes/contrib folder inside the dist version. Its ok if i use deployer/deployer but due to dependency issues i need to use deployer/dist which does not include these folders.

**Reply by antonmedv** [reply]
I DOES include them in a tags. See _dep_ file.
